server/utils/checkYourAnswersUtils.test.ts

```diff
@@ -78,9 +79,12 @@ describe('checkYourAnswersUtils', () => {
 
   describe('getTaskAnswersAsSummaryListItems', () => {
     it('returns an array of summary list items for a given task', () => {
-      jest.spyOn(getQuestionsUtil, 'getQuestions').mockImplementationOnce(jest.fn(() => mockQuestions))
-
-      jest.spyOn(checkYourAnswersUtils, 'getPage').mockReturnValue(jest.fn())
+      const mockQuestionsForOneTask = {
```

Comment:
We redefine the mock questions with just one task, since that's all we care about in this scenario.

Reply:
Not entirely sure this makes this test more readable, having the same questions context for the whole test suite seems fairly easy to grok -- effectively this only duplicates a subset of the existing context test data?

Reply:
yes, avoiding a long and verbose expectation statement. But happy to defer if you want to set it up differently @froddd

Reply:
The expectation statement shouldn't change, given the method under test is called with `task1` specifically. Removing `mockQuestionsForOneTask` and passing `mockQuestions` to the tested method tests the same assertion and doesn't require the expectation to change, which is why I find it a bit odd to duplicate it in this case.

Reply:
ah I see what you mean! yes, sorry Fred, good point and I agree. I've removed it 👍